public/app/core/utils/richHistory.ts

```diff
@@ -5,7 +5,6 @@ import _ from 'lodash';
 import { DataQuery, DataSourceApi, dateTimeFormat, AppEvents, urlUtil, ExploreUrlState } from '@grafana/data';
 import appEvents from 'app/core/app_events';
 import store from 'app/core/store';
-import { SortOrder } from './explore';
```

Comment:
`Sort order` for rich history was moved to `./richHistory` and `Logs sort order` was moved to `grafana/data`.